modules/packages/Sort.chpl

```diff
@@ -274,10 +274,7 @@ proc chpl_check_comparator(comparator, type eltType) {
 /* Basic Functions */
 
 /*
-   General purpose sorting interface.
-
-   .. note:: Currently this method calls a sequential :proc:`quickSort`, but
-             this may change the future as other algorithms are implemented.
+   General purpose sorting interface. This procedure calls :proc:`quickSort`.
```

Comment:
Why should this say it calls quickSort at all? Anyway if it continues to do so, say "currently calls".